{
  try {
    return catchExceptions();
  }
 catch (  Throwable e) {
    return true;
  }
}
